tests/unit/lib.layouts.js

```diff
@@ -0,0 +1,30 @@
+/*global describe, it*/
+
+process.env.NODE_ENV = 'production';
```

Comment:
There are checks on this env variable throughout the code. This just makes things a little easier for me.